### PR TITLE
docs: update CUDA troubleshooting

### PR DESCRIPTION
**Pull request by kenji-miyake**
## Description

Generally, there is no need to uninstall packages.

I've confirmed this [here](https://github.com/autowarefoundation/autoware/pull/154#issuecomment-1091061287).

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by kenji-miyake** [comment]
As @wep21 proposes, we can update the ansible option to prevent this problem, but I think it's good to keep this information here.
https://github.com/autowarefoundation/autoware/pull/154#issuecomment-1091062946

File: docs/help/troubleshooting.md
Comment (xmfcx) [inline docs/help/troubleshooting.md:19]:
```suggestion
    "nvidia*" \
    "python3-libnvinfer"
```

One more thing is held in https://github.com/autowarefoundation/autoware/blob/main/ansible/roles/tensorrt/README.md#manual-installation
`libnvinfer8 libnvonnxparsers8 libnvparsers8 libnvinfer-plugin8 libnvinfer-dev libnvonnxparsers-dev libnvparsers-dev libnvinfer-plugin-dev python3-libnvinfer`

**Review by xmfcx — CHANGES_REQUESTED** [review]
Added one more held package by our installation.

File: docs/help/troubleshooting.md
Comment (kenji-miyake) [reply docs/help/troubleshooting.md:19]: @xmfcx Thank you. But we didn't install the python library with Ansible because we only use C++.
https://github.com/autowarefoundation/autoware/blob/main/ansible/roles/tensorrt/tasks/main.yaml

I think it's good to make the Ansible script and manual installation steps consistent.
Should we add it to the Ansible script or should we remove it from the manual installation steps?

@yukkysaito @wep21 Any thoughts?

Comment (wep21) [reply docs/help/troubleshooting.md:19]: @kenji-miyake I agree with the latter because currently we don't need the python library for autoware.

Comment (kenji-miyake) [reply docs/help/troubleshooting.md:19]: @xmfcx Can I remove the python library from the manual installation steps?

Comment (xmfcx) [reply docs/help/troubleshooting.md:19]: It's part of the default nvidia tensorrt installation process. Why not keep it?

If someone wants to use it, they could get confused.

Comment (xmfcx) [reply docs/help/troubleshooting.md:19]: If you are really worried about the disk-space, you can remove it.

Comment (kenji-miyake) [reply docs/help/troubleshooting.md:19]: > Why not keep it?

It's just because we don't use it now.
If someone wants it in the future, then it's better to review the use case and add it.
I'm worried about some unintended dependencies will be added, it could be troublesome.

But I don't have so strong opinion on this. It's enough if we can make Ansible and Manual consistent.

@xmfcx So considering my answer, which do you prefer? I'll leave it to you.

Comment (xmfcx) [reply docs/help/troubleshooting.md:19]: We can remove it, it's ok.

Comment (kenji-miyake) [reply docs/help/troubleshooting.md:19]: Thank you! Then I'll send a PR to `autowarefoundation/autoware`. After that, please approve this. :pray:

Comment (kenji-miyake) [reply docs/help/troubleshooting.md:19]: https://github.com/autowarefoundation/autoware/pull/177

**Review by xmfcx — APPROVED** [review]

**Pull request merged**
